test-xtradb-operator/test/learn.yaml

```diff
@@ -1,4 +1,5 @@
-mode: learn
+stage: learn
```

Comment:
I remember the learn.yaml can be automatically generated by sieve.py, right?

Reply:
Yes, this learning file is tracked before, maybe I should remove this useless file.